tests/Signature/SignatureProviderTest.php

```diff
@@ -13,6 +13,7 @@ public function versionProvider()
         return [
             ['v4', 'Aws\Signature\SignatureV4', 'foo'],
             ['v4', 'Aws\Signature\S3SignatureV4', 's3'],
+            ['v4-unsigned-body', 'Aws\Signature\SignatureV4', 'foo'],
```

Comment:
No, we are going to switch from things like s3v4 and consolidate them into just v4 and v4-unsigned-body instead.